classes/PrestaShopLogger.php

```diff
@@ -163,7 +169,7 @@ public static function eraseAllLogs()
      *
      * @return true if exists
      */
-    protected function _isPresent()
+    protected function isPresent()
```

Comment:
Hello @firstred 
Can you change it and rebase your branch please ?!
`!$log->_isPresent()` vs `function isPresent()`
Thank you!